src/mds/InoTable.cc

```diff
@@ -111,6 +113,8 @@ void InoTable::replay_alloc_id(inodeno_t id)
 }
 void InoTable::replay_alloc_ids(interval_set<inodeno_t>& ids) 
 {
+  assert(mds);  // Only usable in online mode
+
```

Comment:
Why are we changing the InoTable itself? We don't appear to do anything with it.